cmd/podman/README.md

```diff
@@ -106,8 +86,22 @@ func init() {
 
 // Business logic: cmd is inspectCmd, args is the positional arguments from os.Args
 func inspect(cmd *cobra.Command, args []string) error {
-    // Business logic using registry.ImageEngine
+    // Business logic using registry.ImageEngine()
     // Do not pull from libpod directly use the domain objects and types
     return nil
 }
 ```
+
+## Helper functions
+
+The complete set can be found in the `validate` package, here are some examples:
+
+ - `cobra.Command{ Args: validate.NoArgs }` used when the command does not accept errors
+ - `cobra.Command{ Args: validate.IdOrLatestArgs }` used to ensure either a list of ids given or the --latest flag
+ - `cobra.Command{ RunE: validate.SubCommandExists }` used to validate a subcommand given to a command
+ - `validate.ChoiceValue` used to create a `pflag.Value` that validate user input against a provided slice of values. For example:
+    ```go
+    flags := cobraCommand.Flags()
+    created := validate.ChoiceValue(&opts.Sort, "command", "created", "id", "image", "names", "runningfor", "size", "status")
+    flags.Var(created, "sort", "Sort output by: "+created.Choices())
+    ```
```

Comment:
Might be good to include compilation instructions and how to do vendoring.  And/or point to the contributors.md for more information.